# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gnokii/hotfix-static.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gnokii-0.6.31.orig/common/gsm-filetypes.c	2011-12-02 15:53:04.000000000 +0100
+++ gnokii-0.6.31/common/gsm-filetypes.c	2026-07-26 06:35:34.043540460 +0200
@@ -1409,7 +1409,7 @@
 #define GET_NEXT_TOKEN()	o = get_next_token(line + offset, ';')
 #define STORE_TOKEN(a)		strip_slashes(a, line + offset, sizeof(a) - 1, o - 1)
 
-inline int local_atoi(char *str, int len)
+static inline int local_atoi(char *str, int len)
 {
 	int retval;
 	char *aux = strndup(str, len);
